### ISSUE TITLE
Quality of life: powershell script for automation (already implemented)

### ISSUE THREAD

**Issue by ikrima** [Viladoman/CompileScore#3]
## TLDR
Powershell script for automating captures https://gist.github.com/ikrima/45497f8d664c35f0d656b95773426ac3

Not sure if this is useful enough to be in the repo. If so, lmk, I can make a PR, update the defaults, and strip references to CppBuildAnalyzer

## Details
I got tired of starting/stopping and made a powershell script that auto-elevates. Also I always forget the steps after a week and end up hunting them down

The gist auto elevates but is annoying that the UAC prompt will come up each time. 
My local version uses gsudo which caches elevation requests for 5 mins but the repo version probably shouldn't use it bc security
The two lines are just commented out in the gist

Running it is simple:

```powershell
.\vcperf.ps1 -start
[build in visual studio]
.\vcperf.ps1 -stop -compilescore
```

**Comment by Viladoman** [comment]
I liked it. Thanks for sharing the script. 
Imho, the admin rights for getting build insights in MSVC is really annoying so the easier we can make this process the better. 
I don't mind having a folder with helpers/utils in the repo. Feel free to do the PR or we can add the link to the Gist in the readme page.
